src/test/java/com/mongodb/jdbc/BsonTypeInfoTest.java

```diff
@@ -0,0 +1,36 @@
+package com.mongodb.jdbc;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.sql.SQLException;
+import org.junit.jupiter.api.Test;
+
+public class BsonTypeInfoTest {
+    @Test
+    void testGetBsonTypeInfoByName() throws SQLException {
```

Comment:
A test with an unexpected bsonName would be good to check that it throws an exception.

Reply:
Good idea, will add!